packages/css-data/src/__generated__/keyword-values.ts

```diff
@@ -3836,7 +3836,7 @@ export const keywordValues = {
     "unset",
   ],
   fontWeight: [
-    "normal",
+    "400",
     "bold",
     "bolder",
     "lighter",
```

Comment:
There's also bolder/lighter that we may need to handle above

Reply:
Ah, yeah. didn't notice. Need to revert this one

Reply:
Fixed bold. Though bolder and lighter are relative. Will leave as is. We anyway do not use this list. Possible weights are extracted from font.